libs/README.md

```diff
@@ -14,4 +14,27 @@
 
 # Libraries
 
-Contains the source code for Block Protocol development libraries, utilities, and plugins. Full write-ups of most can be found in the [Block Protocol docs], and a summary table will shortly be published below.
+Contains the source code for Block Protocol development libraries, utilities, and plugins. Full write-ups of most can be found either in `README.md` files in each individual library folder or in the [Block Protocol docs], and a summary table will shortly be published below.
+
+## Publishing
+
+All libraries except those in `@local/` are published to `npm` via
+[Changesets](https://github.com/changesets/changesets).
+
+To record a change for publication:
+
+1.  From the root of the repository, run `yarn changeset`
+1.  Select the package(s) affected by this change (space to select, enter to move to the next step)
+
+- Do not worry about selecting packages which depend on changed packages – Changesets will handle bumping them
+
+1.  Select the semver increment
+1.  Describe the change
+1.  Commit the created changeset file
```

Comment:
Nit: the formatting for these numbered bullets is going to be slightly off, as they will start counting from `1.` again mid-way down the list. Maybe worth using `3.`/`4.`/`5.` explicitly in this use-case
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/42802102/230036672-3d1309f2-a8a3-43f1-a9c9-cb8a8bce8d71.png">

Reply:
I think explicitly using them will cause markdownlint errors. I _think_ we want to solve this by actually nesting the `-`

Reply:
Done (requiref 4 spaces)